config/initializers/loa.rb

```diff
@@ -5,10 +5,4 @@ module LOA
   ONE   = 1
   TWO   = 2
```

Comment:
Do we want to keep a module just to map words to numbers? I'm all for constants for more complex things, but this seems like overkill.

Reply:
LOL, i'll get rid of this eventually, but right now it would require changing too many files in this PR. I got rid of have of that file in this one though. :)

The only benefit I see is that in specs and elsewhere it's kind of more readable and searchable. It's easy to search for LOA::ONE, but searching for '1' not so easy.

Reply:
good point on search ability.